docker-compose/compose.yaml

```diff
@@ -43,14 +43,15 @@ services:
       start_period: 30s
 
   exporter:
-    image: container-registry.oracle.com/database/observability-exporter:1.2.0
+    image: container-registry.oracle.com/database/observability-exporter:1.3.0
     container_name: exporter
     ports:
       - 9161:9161
     environment:
-      - DB_USERNAME=system
+      - DB_USERNAME=sys
       - DB_PASSWORD=Welcome12345
```

Comment:
DO we want an actual password here?

Reply:
it has to have one to start it up :)

Reply:
Yeah but maybe say <UOOUR_PASSWORD_HERE>